src/waveform/renderers/allshader/waveformrendererpreroll.cpp

```diff
@@ -20,7 +21,7 @@ QImage drawPrerollImage(float markerLength,
     const float imageW = static_cast<float>(imagePixelW) / devicePixelRatio;
     const float imageH = static_cast<float>(imagePixelH) / devicePixelRatio;
 
-    QImage image(imagePixelW, imagePixelH, QImage::Format_ARGB32_Premultiplied);
+    QImage image(imagePixelW, imagePixelH, QImage::Format_RGBA8888_Premultiplied);
```

Comment:
can you quickly explain your rationale behind this?
According to the QImage docs:
> Note: Avoid most rendering directly to most of these formats using [QPainter](https://doc.qt.io/qt-6/qpainter.html). Rendering is best optimized to the Format_RGB32 and Format_ARGB32_Premultiplied formats, and secondarily for rendering to the Format_RGB16, Format_RGBX8888, Format_RGBA8888_Premultiplied, Format_RGBX64 and Format_RGBA64_Premultiplied formats

Reply:
I don't remember. Let me try changing it back and see if something goes wrong :-)

Reply:
ARGB32 works fine, so no idea why I changed this...